test/nginx_system_test.sh

```diff
@@ -289,7 +289,6 @@ SERVER_NAME=nginx
 
 # run generic system tests
 SYSTEM_TEST_FILE="$MOD_PAGESPEED_DIR/src/pagespeed/system/system_test.sh"
-AUTOMATIC_SYSTEM_TEST="$MOD_PAGESPEED_DIR/src/net/instaweb/automatic/system_test.sh"
```

Comment:
We used to need to tell `system/system_test.sh` how to find `automatic/system_test.sh`, but now that `automatic/` and `/system` are in sibling directories `system/system_test.sh` can do it itself and ignores the `AUTOMATIC_SYSTEM_TEST` variable.